src/ensembl/production/metadata/admin/admin.py

```diff
@@ -0,0 +1,245 @@
+#   See the NOTICE file distributed with this work for additional information
+#   regarding copyright ownership.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.from django.apps import AppConfig
+
+
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+from ensembl.production.metadata.admin.filters import *
+
+
+class AssemblySequenceInline(admin.StackedInline):
+    model = AssemblySequence
+    fields = ['name', 'length']
+    can_delete = False
+    can_update = False
+
+    def has_add_permission(self, request, obj):
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
+
+@admin.register(Assembly)
+class AssemblyAdmin(admin.ModelAdmin):
+    read_only_fields = ('created',)
+    fields = ['accession', 'name', 'ucsc_name', 'accession_body', 'level', 'assembly_default']
+    list_filter = ('accession',)
+    search_fields = ('accession',)
+    order = ('accession',)
+    inlines = (AssemblySequenceInline,)
+
+    def get_readonly_fields(self, request, obj=None):
+        read_only_fields = super().get_readonly_fields(request, obj)
+        if request.user.is_superuser:
+            read_only_fields += ('name',)
+        return read_only_fields
+
+
+#####RELEASE ADMIN PAGE#####
+
+class GenomeReleaseInLine(admin.TabularInline):
+    model = GenomeRelease
+    fields = ['Assemblies', 'Organisms', 'Datasets']
+    readonly_fields = ["Assemblies", 'Organisms', 'Datasets']
+    can_delete = False
+    can_update = False
+
+    def Assemblies(self, obj):
```

Comment:
Lower case and without plural, you are retrieving only one line here, right? 
```suggestion
    def genome_assembly(self, obj):
```